server/templates/base.html

```diff
@@ -2,7 +2,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    {% block head %}{% endblock %}
```

Comment:
I don't think we want to remove these block directives, they are necessary to be able to vary the inheritance between the editor and other server-side pages (the editor needs different css directives e.g.)